bsdlib/include/nrf_socket.h

```diff
@@ -275,12 +275,27 @@ typedef uint32_t nrf_fd_set;
  * Identifies the option used to set the GNSS fix interval in seconds.
  *
  * @details
+ * Fix interval parameter controls the time between GNSS receiver starts.
+ * It also controls whether the GNSS receiver is stopped after a valid PVT
+ * estimate. The fix interval parameter determines the mode of navigation.
+ * There are three different navigation modes available: single-fix,
+ * continuous, and periodic.
+ *
  * Single-fix navigation mode is engaged by setting the fix interval to 0.
+ * In this mode once a valid PVT estimate is produced, the GNSS receiver is
```

Comment:
Please do not describe the modes here, this information is in https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/bsdlib/doc/gnss_extension.html

Reply:
same